frontend/ext/varfish-api/src/lib/types.gen.ts

```diff
@@ -1985,25 +1957,11 @@ export type SeqvarsQuerySettingsDetails = {
  * Serializer for ``QuerySettingsFrequency``.
  */
 export type SeqvarsQuerySettingsFrequency = {
-    gnomad_exomes_enabled?: boolean;
-    gnomad_exomes_frequency?: number | null;
-    gnomad_exomes_homozygous?: number | null;
-    gnomad_exomes_heterozygous?: number | null;
-    gnomad_exomes_hemizygous?: boolean | null;
-    gnomad_genomes_enabled?: boolean;
-    gnomad_genomes_frequency?: number | null;
-    gnomad_genomes_homozygous?: number | null;
-    gnomad_genomes_heterozygous?: number | null;
-    gnomad_genomes_hemizygous?: boolean | null;
-    helixmtdb_enabled?: boolean;
-    helixmtdb_heteroplasmic?: number | null;
-    helixmtdb_homoplasmic?: number | null;
-    helixmtdb_frequency?: number | null;
-    inhouse_enabled?: boolean;
-    inhouse_carriers?: number | null;
-    inhouse_homozygous?: number | null;
-    inhouse_heterozygous?: number | null;
-    inhouse_hemizygous?: number | null;
+    gnomad_exomes?: unknown;
+    gnomad_genomes?: unknown;
+    gnomad_mitochondrial?: unknown;
+    helixmtdb?: unknown;
+    inhouse?: unknown;
```

Comment:
@holtgrewe it looks like type introspection did not work here

Reply:
Sorry @Gregoor Will fix on Wednesday. Will be the equivalent of

https://github.com/varfish-org/varfish-server/blob/3c325fe892edde0a37da9030418392018439c93b/backend/seqvars/models.py#L21

Reply:
Should work after https://github.com/varfish-org/varfish-server/pull/1786

Reply:
Thank you!